Makefile

```diff
@@ -49,7 +49,11 @@ documents.js: bin/build_search_documents.py $(XMLS)
 #
 .SECONDEXPANSION:
 %.html: $$(dir $$@)text.xml devbook.xsl xsl/*.xsl $$(subst text.xml,index.html,$$(wildcard $$(dir $$@)*/text.xml))
+ifdef FALLBACK
```

Comment:
I accidentally ran `make FALLBACK=0 clean all` and realized it's still passing `--param fallback 1`. How about something like:

```
diff --git a/Makefile b/Makefile
index 81aab5c..5b2d660 100644
--- a/Makefile
+++ b/Makefile
@@ -6,6 +6,7 @@ SVGS := $(shell find . -name .git -prune -o -type f -name '*.svg' -print)
 HTMLS := $(subst text.xml,index.html,$(XMLS))
 ECLASS_HTMLS := $(wildcard eclass-reference/*/index.html)
 IMAGES := $(patsubst %.svg,%.png,$(SVGS))
+FALLBACK ?= 0
 
 all: prereq validate build documents.js
 
@@ -49,11 +50,7 @@ documents.js: bin/build_search_documents.py $(XMLS)
 #
 .SECONDEXPANSION:
 %.html: $$(dir $$@)text.xml devbook.xsl xsl/*.xsl $$(subst text.xml,index.html,$$(wildcard $$(dir $$@)*/text.xml))
-ifdef FALLBACK
-       xsltproc --param fallback 1 devbook.xsl $< > $@
-else
-       xsltproc devbook.xsl $< > $@
-endif
+       xsltproc --param fallback $(FALLBACK) devbook.xsl $< > $@
 
 validate: prereq
        @xmllint --noout --dtdvalid devbook.dtd $(XMLS) \
```

Would it bother you that it's unconditionally passing the parameter on the command line?

Reply:
> Would it bother you that it's unconditionally passing the parameter on the command line?

Yeah, I didn't like it, but I guess there's no real technical reason not to do it as you suggested.